pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -13914,7 +13914,6 @@ self: {
        description = "A minimalistic CLI Pomodoro timer";
        license = lib.licenses.mit;
        hydraPlatforms = lib.platforms.none;
```

Comment:
This basically looks good to me.

Here's a couple suggestions for future PRs:

- I'm guessing you manually edited this `hackage-packages.nix` file?  In general, this `hydraPlatforms` line also needs to be removed in order for Hydra to build your package.  You can do this by running `/maintainers/scripts/haskell/regenerate-hackage-packages.sh` instead of editing this file by hand.
- We generally only merge haskell-related PRs into the `haskell-updates` branch.  So in future PRs, make sure to base your PR on this branch.  I'll do this right now before merging in.
- Running ofborg yourself makes it a little easier for someone reviewing your PR to confirm that the package actually builds.

Reply:
Thanks for the suggestions!